src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Square extends Quadrangle {
```

Comment:
Don't use verbs for class/interface names. Classes should be named after what they represent, and since 'Square' is a noun, it's correctly named.